src/helpers/handleError.ts

```diff
@@ -32,11 +35,14 @@ export function parseErrorText(
   if (message) {
     const gSNMessage = parseGSNError(message)
     const revertMessage = parseRevertReason(message)
-    if (/cannot estimate gas/.test(message) && !revertMessage && !gSNMessage) {
-      displayedError = ErrorList.invalidProof
-    } else {
-      displayedError = gSNMessage || revertMessage || message
-    }
+    const rejectSignatureMessage = parseRejectSigningError(message)
+    const invalidProofMessage = parseInvalidProofError(message)
+    displayedError =
```

Comment:
Instead of multiple parsers lets do something like this
![IMG_20221004_132958](https://user-images.githubusercontent.com/49658988/193797705-c82eca68-1a79-4b38-adbd-7ee7ff60ad3b.jpg)

Reply:
Yeah, I wasn't home, so wrote you this :)

Reply:
Using `Object.keys` just converts the object into keys only and the error itself can't be accessed. I do like this approach tho, so I'll see if I can come up with anything that's similar

Reply:
You can use `Object.entries` or access the list by `errors[errKey]`